src/Psalm/Internal/CallMap.php

```diff
@@ -354,13 +354,13 @@
 'ArithmeticError::getLine' => ['int'],
 'ArithmeticError::getMessage' => ['string'],
 'ArithmeticError::getPrevious' => ['?Throwable'],
-'ArithmeticError::getTrace' => ['array<int,array<string,mixed>>'],
+'ArithmeticError::getTrace' => ['list<array<string,mixed>>'],
 'ArithmeticError::getTraceAsString' => ['string'],
 'array_change_key_case' => ['array|false', 'input'=>'array', 'case='=>'int'],
 'array_chunk' => ['list<array[]>', 'input'=>'array', 'size'=>'int', 'preserve_keys='=>'bool'],
 'array_column' => ['array', 'array'=>'array', 'column_key'=>'mixed', 'index_key='=>'mixed'],
 'array_combine' => ['array|false', 'keys'=>'string[]|int[]', 'values'=>'array'],
-'array_count_values' => ['int[]', 'input'=>'array'],
+'array_count_values' => ['array<string|int, int>', 'input'=>'array'],
```

Comment:
Please revert this change – various tools (including Psalm) treat `array<int>` and `array<string|int, int>` slightly differently.

Reply:
Done. Pretty curious what is treated differently and why though. int[] looks like missing details to me. Even `array-key` is ambiguous and doesn't say much more about actual type.

If it has any incidence in user land annotated code, it would be nice to know what to expect :)

Thanks!

Reply:
`array-key` is treated slightly like `mixed` – it's a "not sure" type. `string|int` means "it can either be a string or an int – either is valid". 

So `array-key` is to `string` and `int` what `mixed` is to every sort of type – a less-specific version thereof.